test/apps/webgpu-point-cloud/app.tsx

```diff
@@ -0,0 +1,88 @@
+// deck.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+import React from 'react';
```

Comment:
To avoid creating another test app, can you add this layer into `test/apps/webgpu-scatterplot/app.tsx`?